### ISSUE TITLE
Recipe Clash - Artisanat 2.2.1

### ISSUE THREAD

**Issue by UntalentedAmateur** [DawnTeamMC/Artisanat#24]
### What happened?

There is a recipe clash for the use of various *_WOOD blocks.  In vanilla MC, a single wood block is convertable to 4 planks, the same as the corresponding *_LOG block.

With Artisanat 2.2.1 (earlier versions have not been tested) the .json recipe files for the various *_wood_button options are based on a single corresponding wood block.  Unfortunately, this stops the default MC recipe being available, and additional mods like REI and polymorph (fabric versions) can't seem to show the default recipe and trying to craft it as normal only offers the Artisanat button as the recipe option.  As there is no recipe to reverse _WOOD back into _LOG, once you have a wood block, you're stuck with the Artisanat recipe pathway.

A suggestion would be to have the Artisanat button recipes more closely match those of the default MC, where one plank generates a button.  From looking at the files within the JAR, it seems that it would be a relatively straightforward modification of the corresponding wood button JSON in:

- /data/artisanat/advancements.recipes/crafting
- /data/artisanat/recipes/crafting

Because this modified recipe is still using extant blocks and crafting patterns, this shouldn't impact any code deeper in the .class files (at least I haven't found anywhere else it seems to be present).

I might fork and test a local build with these changes and, if they work well, will push them back as a PR.

### Dawn API version

3.3.0

### Fabric API version

0.51.1+1.18.2

**Comment by Hugman76** [comment]
It doesn't make sense that the bark of the wood would come up after it was transformed into planks, though.

**Comment by UntalentedAmateur** [comment]
> It doesn't make sense that the bark of the wood would come up after it was transformed into planks, though.

No.  Not really.  There are some environmental mods that change the appearance of some trees to wood vice logs, and it's also about bringing back more of the vanilla behaviour of being able to use wood-> planks transformation.  I can't see much / any other way to retain the vanilla recipe and still have the Artisanat recipe as well.

Maybe it can be thought of as a live edge plank, and we're creating the buttons from the live edges, hence the retained bark features.

**Comment by UntalentedAmateur** [comment]
> Heh, I think that it breaks the consistency of wood -> wood blocks and planks -> planks blocks. Plus, I don't think that it's good idea either to replace a conflicting recipe with another conflicting recipe

True.  Kind of a difficult situation, but at least the vanilla recipes are working again.

PR created (https://github.com/DawnTeamMC/Artisanat/pull/26).  Closing, with Hugman to determine future behaviour.

Thanks for the awesome set of Dawn related mods!

**Comment by Hugman76** [comment]
Let's leave the issue open until it's actually fixed

**Comment by UntalentedAmateur** [comment]
I've put some alternative recipe options in the PR comments, which might better preserve your intent.

I'm not sure what more I can contribute at this point.

**Comment by Hugman76** [comment]
Concerning this issue, I now believe there is no way around it rather than removing the Wood-to-button recipe and adding a recipe either in stonecutter or new "woodcutter" block.
